src/advanced/node-management.md

```diff
@@ -755,6 +755,64 @@ equal to `start` and is less than `end`.
     That is, to collect `5` non-empty blocks from the tail of the blockchain,
     range from `288` to `6` has been traversed.
 
+## Explorer API Sockets
+
+Since Exonum 0.10 version, it is possible to connect to nodes via WebSocket.
+
+Explorer API sockets have the same base path as endpoints, denoted
+**{explorer_base_path}** and equal to `/api/explorer/v1`.
+
+Currently only one socket is implemented - it shares information on block
+commit events.
+
+### Subscribe to Block Commit
+
+```none
+ws://${URL}{explorer_base_path}/blocks/subscribe
+```
+
+Connects to a socket and receives notices on each new committed block starting
+from the moment of connection. The notices are displayed in the blockchain
+explorer.
+
+#### Parameters
+
+None.
+
+#### Response
+
+Returns notifications that a new block has been committed to the blockchain
+starting from the height when the client connected to the socket.
+
+Each notification is a JSON object with the following fields:
```

Comment:
`Each notification is a string witch could be deserialized into JSON object which contains the following fields:`

Reply:
done